draft-ietf-rtcweb-jsep.xml

```diff
@@ -276,9 +276,11 @@ ipr="trust200902">
 
         <t>In addition, various RFCs put different conditions on the
         format of offers versus answers. For example, an offer may
-        propose an arbitrary number of media streams (i.e. m=
-        sections), but an answer must contain the exact same number as
-        the offer.</t>
+        propose an arbitrary number of media streams (i.e., media
+        descriptions as described in <xref target="RFC4566"/>, Section
+        5.14) but an answer must contain the exact same number as
+        the offer. Note: this document uses the terms "m= section",
```

Comment:
This feels like an odd place for this note, as it's largely irrelevant to this section. 

I suggest mentioning this in the "Session Description Format" section below.